service/history/workflow/mutable_state_impl.go

```diff
@@ -2853,6 +2854,9 @@ func (ms *MutableStateImpl) ApplyActivityTaskScheduledEvent(
 	firstEventID int64,
 	event *historypb.HistoryEvent,
 ) (*persistencespb.ActivityInfo, error) {
+	// QQQQQQQQ
+	println(fmt.Sprintf("ApplyActivityTaskScheduledEvent, %v", time.Now().UTC()))
```

Comment:
You're going to remove this before merging, right?

Reply:
removed.